packages/page-staking/src/Suspensions/CurrentList.tsx

```diff
@@ -43,10 +46,18 @@ function CurrentList ({ suspensions }: Props): React.ReactElement<Props> {
     '5H9h84SrX4gdXTxGyB6wtEfTye5Kb7vMcwARNLCZxMa1CruS', '5HdUiPkneLL2dQHvFkp47cfw63uPWSL8gFPzduLw6YXx3cBU'
   ];
 
+  const filteredSuspensions = suspensions?.filter(
```

Comment:
Nit: I'm generally in favour of putting logic like that in a separate function. Purely a stylistic choice, though ;)

Reply:
I agree in general, but in this case there are so many variables from the clojure, that the function would make it less readable than a direct instruction, so I'll leave it this time.